.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,5 @@
-<!-- Please review the following before submitting a PR:
+<!-- 
+Please review the following before submitting a PR:
 osu!'s Article Styling Criteria: https://osu.ppy.sh/help/wiki/Article_Styling_Criteria

```

Comment:
if you kept the checklist section, you can remove this comment and put this in the checklist (this was what I was talking about for the checklist section)

```
### Checklist

- [ ] Reviewed [ASC](https://osu.ppy.sh/wiki/ASC)
```

Reply:
No real point for that kind of checklist atm, just making it sleekier and simple for now